### DISCUSSION TITLE
Wifi 6 (802.11ax)

### DISCUSSION THREAD

**Discussion by slavkobrzeg** [OpenEVSE/open_evse#35] [General]
Hi awesome guys!
I'm using my OpenEVSE kit since 2020. It is on old ESP, not easily upgradable.... but for me it works like a harm since then! I'm thinking about to buy the second unit and I love what I can see now, where upgrades went. 

But I'm wondering if EVSE uses ESP chip with a support for WiFi 6 (802.11ax) or not yet? And if no, how difficult would it be to replace the ESP board. Wifi ax is awesome. It improved the coverage in my location a lot and I'm trying not to buy new devices without wifi 6 on board, because I want to use them for years and wifi ax will be a standard very soon.

**Comment by tekym** [comment]
You can absolutely upgrade the wifi module from the old ESP8266 to ESP32.  I did that not too long ago to my 2016-era OpenEVSE, referring to the charging stations assembly directions for both 2016 as well as the current version, so I could make sure the wiring connected correctly.  Looks like you might not even need to do that, 2020 might be the same version as current, so it'd be plug and play.

I can't imagine there's much reason to make an OpenEVSE use bleeding-edge 802.11ax at this point, it's not a heavy data source.  All wifi devices are backwards compatible anyway,

**Reply by slavkobrzeg** [reply]
@tekym THanks for your insight! And yes, I am thinking about upgrading my '20 EVSE with the new ESP board. All I probably need is to map all PINs correctly and flash it. But someone said: "If it works, don't change it!". 
I was asking about wifi 6 because the other location I'm planning to use the second item has a very specific environment and there is a weak coverage there on 2.4 (n) and simply 2.4ax would be much better. More stable. I tested that. So having EVSE with wifi 6 will save me some money on extending the wifi coverage. 
I know, I know... this or that way I will have some extra spending, but I'm asking about it, because again this is only a case to replace the ESP bard with the newer one equipped wifi wifi 6 too. 
Maybe even the same way as my 2020 I can upgrade it on my own :)

**Comment by cmgj74** [comment]
Here is a good deal on the wifi...ymmv

https://www.ebay.com/itm/145300530776
